src/Hypergrid.js

```diff
@@ -1261,22 +1261,6 @@ var Hypergrid = Base.extend('Hypergrid', {
         return window.open.apply(window, arguments);
     },
 
-    /**
-     * @memberOf Hypergrid#
-     * @desc Synthesize and dispatch a `fin-selection-changed` event.
-     */
-    selectionChanged: function() {
```

Comment:
Nice. Good to keep this file as lean. Even if it will still have a million API calls on it. Someday we should do the Hyperblotter revisit and API reshaping